src/timesync/timesyncd-manager.c

```diff
@@ -383,10 +383,10 @@ static int manager_adjust_clock(Manager *m, double offset, int leap_sec) {
                   "  offset       : %+.3f sec\n"
                   "  freq offset  : %+li (%i ppm)\n",
                   tmx.status, tmx.status & STA_UNSYNC ? "unsync" : "sync",
-                  tmx.time.tv_sec, tmx.time.tv_usec / NSEC_PER_MSEC,
-                  tmx.constant,
+                  (long int)tmx.time.tv_sec, tmx.time.tv_usec / NSEC_PER_MSEC,
```

Comment:
Retracting the comment about tv_usec needing a cast. NSEC_PER_MSEC is already defined as 1000000ULL, so that division is always promoted to `unsigned long long`, which might or might not be compatible to `PRI_USEC` which matches `uint64_t`. The comment about using `PRI_TIME` instead of casting a time_t variable (`tv_sec`) still stands, though.

Using NSEC_PER_MSEC to convert tv_**u**sec to milliseconds smells fishy, though, but is unrelated to inaccurate format strings.